### PR TITLE
Bugfix/164 ui fixes

### LINKED ISSUE #164: Business partner menu still scroll
It looks like the problem is still there:

image

Also additionally, Tom pointed out that both overlays are not vertically aligned which I think is related to this problem as well

### PR DESCRIPTION
**Pull request by frontendherodk**
Fixes #164 #167

### CODE REVIEW

File: src/app/common/visibility/visibility.service.spec.js [hunk 1]
```diff
@@ -91,7 +91,23 @@ describe('Visibility Service', () => {
 
       expect(stateHandlerService.dispatch).toHaveBeenCalledWith(mock.dispatchAction);
     });
-  })
+  });
+
+  describe('body', () => {
```
Comment (dami-gg) [inline src/app/common/visibility/visibility.service.spec.js]: I'll do these kind of small changes on the PR directly but for the records: it's better to give the name of the method you're testing to the describe block, in this case toggleBodyClass

**Comment by dami-gg** [comment]
Nice way to fix it, it looks fine to me!

**Comment by dami-gg** [comment]
:+1:

**Comment by frontendherodk** [comment]
👍

**Pull request merged**